### PR TITLE
[Data Mapper] Expanded function cards

### PR DESCRIPTION
**Pull request by refortie**
image

image

### CODE REVIEW

File: libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx [hunk 1]
```diff
@@ -77,9 +77,13 @@ const OverviewReactFlowWrapper = () => {
     fitView(reactFlowFitViewOptions);
   }, [fitView, sourceSchema?.schemaTreeRoot.key, targetSchema?.schemaTreeRoot.key]);
 
+  const schemaNodeType = {
```
Comment (NicL9923) [inline libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx:80]: Move this outside the FC or memoize it where it is so ReactFlow doesn't get mad at you

Comment (NicL9923) [reply libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx:80]: *IIRC

File: libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx [hunk 2]
```diff
@@ -69,6 +68,15 @@ export const ReactFlowWrapper = ({ canvasBlockHeight, canvasBlockWidth }: ReactF
...
   const [displayMiniMap, { toggle: toggleDisplayMiniMap }] = useBoolean(false);
 
+  const nodeTypes = useMemo(
+    () => ({
+      [ReactFlowNodeType.SchemaNode]: SchemaCard,
+      [ReactFlowNodeType.FunctionNode]: useExpandedFunctionCards ? ExpandedFunctionCard : SimpleFunctionCard,
+    }),
+    [useExpandedFunctionCards]
+  );
+  const edgeTypes = { [ReactFlowEdgeType.ConnectionEdge]: ConnectionEdge };
```
Comment (NicL9923) [inline libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx:78]: Assuming the same case as previous comment

**Review by NicL9923 — APPROVED** [review]

File: libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx (hunk 1 above)
Comment (refortie) [reply libs/data-mapper/src/lib/components/mapOverview/MapOverview.tsx:80]: I believe you are correct. Changed

File: libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx (hunk 2 above)
Comment (refortie) [reply libs/data-mapper/src/lib/ui/ReactFlowWrapper.tsx]: Changed

**Pull request merged**